v2/interface.go

```diff
@@ -61,6 +61,8 @@ type ClientConfiguration struct {
 	// any request parameters or request or response fields that correspond to
 	// alpha features.
 	EnableAlphaFeatures bool
+	// CAData holds PEM-encoded bytes (typically read from a root certificates bundle).
```

Comment:
Probably worth a note here that if CAData is set on the `TLSConfig` field, that this will override that value.